tests/zfs-tests/include/libtest.shlib

```diff
@@ -279,6 +279,10 @@ function default_setup_noexit
 	typeset no_mountpoint=$4
 	log_note begin default_setup_noexit
 
+	# Crudely log to dmesg which test we're running for post-mortem
+	me=`ps a | grep -Eo '[^ ]+ksh$' | head -n 1`
+	echo "ZTS: running $me" > /dev/kmsg
```

Comment:
this is very Linux-specific are you sure it isn't easier and more portable to use `logger` ?

Reply:
The intent here is to see the test names interspersed with kernel messages in dmesg so I can see where which test is causing the `__kernel_write()` error in #10940.  I don't think `logger` can log to the dmesg log.